docs/low-code-jobs/databricks-jobs.md

```diff
@@ -3,23 +3,165 @@ sidebar_position: 1
+| Schedule Interval         | Defines how often your job is going to run. The interval is defined using the [Quartz format](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html). You can click on the clock icon, to easily pick the interval. |
+| Pipeline level Timeout    | Timeout at Pipeline level                                                                                                                                                                                                                           |
+| Alerts Email for Pipeline | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for entire Pipeline.                                                                                         |
+| Per Gem Timeout           | Timeout for each Gem in job Pipeline                                                                                                                                                                                                                |
+| Number of retries per Gem | Number of retries for each Gem in job Pipeline                                                                                                                                                                                                      |
+| Alerts Email per Gem      | Comma separated list of emails, that are going to receive notifications on specific job status events (job start, failure, or success) for each Gem in job Pipeline.                                                                                |
+
 ## Deployment
 
+To deploy a job on Databricks, we need to release the project from Prophecy UI as shown in example below. As soon as the project is
+released, the job would start appearing on Databricks jobs page as well.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://user-images.githubusercontent.com/103921419/184726064-67e3df01-ba4c-431e-92e9-8bda92a19530.mp4" title="Job Deployment" allow="autoplay;fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+
+---
+
+:::info
+
+Please ensure to enable the job before release on the job page for it to run on the specified schedule. If it is not enabled, then the job would get released but with paused state.
+
+If fabric is changed in the job, then it would create a new job in the Databricks jobs page, but the job with older fabric would be paused automatically and the new job would be scheduled.
```

Comment:
Phrasing: If a Job's selected Fabric is changed it will create a separate Databricks job definition. The previous Job (with the previous Fabric) will be paused automatically and the new version will be scheduled.